contrib/images/proto-builder/Dockerfile

```diff
@@ -2,15 +2,11 @@
 # docker build --pull --rm -f "contrib/devtools/Dockerfile" -t cosmossdk-proto:latest "contrib/devtools"
 # docker run --rm -v $(pwd):/workspace --workdir /workspace cosmossdk-proto sh ./scripts/protocgen.sh
 
-FROM bufbuild/buf:1.1.0 as BUILDER
+FROM bufbuild/buf as BUILDER
 
-FROM golang:1.18-alpine
+FROM faddat/archlinux
```

Comment:
does this image needs changing? Seems fine here since this isn't used anywhere other than proto generation

Reply:
The idea is to move down to a single image that we use everywhere, so that there are fewer layers.  We need to bootstrap, though.  This is one of many urls that would change to something like ghcr.io/cosmos/cosmos-sdk or such.